EpikCord/__main__.py

```diff
@@ -1,4 +1,60 @@
-from EpikCord import __version__
+import argparse
+import colorama
+
+
+__all__ = ("__version__", "info")
+__version__ = "0.5.2"
+_bot_tmplate = """#!/usr/bin/env python
+from EpikCord import Client, Intents
+
+intents = Intents.all()
+
+client: Client = Client([token], intents)
+
+
+@client.event()
+async def ready():
+    print("Ready!")
+# Write commands here
+
+client.login()
+"""
+win_restricted = (
+    "CON",
+    "CON1",
+    "CON2",
+    "CON3",
+    "CON4",
+    "CON5",
+    "CON6",
+    "CON7",
+    "CON8",
+    "CON9",
+    "AUX",
+    "AUX1",
+    "AUX2",
+    "AUX3",
+    "AUX4",
+    "CON5",
+    "CON6",
+    "CON7",
+    "CON8",
+    "CON9",
+
+)
+
+parser = argparse.ArgumentParser()
+parser.add_argument(
+    "--version", help="displays the version of the library", action="store_true"
+)
+colorama.init(autoreset=True)
+
+
+def setup():
+    print(f"{colorama.Back.RED}{colorama.Fore.BLUE}Welcome To Setup!")
```

Comment:
That combination of color poorly accessible, and on some config it could be pretty much impossible to read the text
![image](https://user-images.githubusercontent.com/53050011/180611266-efa830ca-b37c-44a1-b024-544c3f67a205.png)